engines/gob/save/saveload_v3.cpp

```diff
@@ -38,7 +38,7 @@ SaveLoad_v3::SaveFile SaveLoad_v3::_saveFiles[] = {
 	{  "intro.$$$", kSaveModeSave  , nullptr, "temporary sprite"},
 	{   "bloc.inf", kSaveModeSave  , nullptr, "notes"},
 	{   "prot",     kSaveModeIgnore, nullptr, nullptr},
-	{ "config",     kSaveModeIgnore, nullptr, nullptr},
+	{ "config",     kSaveModeIgnore, nullptr, "configuration"},
```

Comment:
Why/how are you making this change? What does it affect?

Reply:
i thought since its an config file we can add it there with the name. but may we do not need this change?
For the future i would ask when we use the names and when we use in those cases nullptr?

If its its an not required change we can exclude it from the PR if the other commits are OK.

Reply:
@BJNFNE please do not modify code if you are not sure of what you're doing and if you haven't tested the change. The impact of this would be to invalidate saved games, and most probably break existing functionality.

Please revert

Reply:
reverted

Reply:
In fact, this change seems completely useless as the SaveFile entry has a SaveModeIgnore save mode.